rasa/utils/tensorflow/model_data_utils.py

```diff
@@ -63,8 +68,20 @@ def featurize_training_examples(
                     attribute, featurizers
                 )
         output.append(attribute_to_features)
-
-    return output
+    # get sparse feature sizes
+    sparse_attributes = []
+    for attr, features in output[0].items():
+        if features and features[0].is_sparse():
+            sparse_attributes.append(attr)
+    # we exclude labels at this point
+    if label_attribute in sparse_attributes:
+        sparse_attributes.remove(label_attribute)

```

Comment:
Lets remove the logic of filtering out label attribute outside this function. It adds a weird coupling of two different features for which the function is needed.